contrib/clojure-package/README.md

```diff
@@ -193,8 +193,8 @@ The outcome of this step will be a shared library `lib/libmxnet.so` that is used
 - Build and install the Scala package in your local Maven directory using the following commands:
 
   ```bash
-  make scalapkg
-  make scalainstall
+  cd scala-package
+  mvn install
```

Comment:
I would rather make is : ```mvn clean install```

Reply:
As a new user tutorial, I think it should be fine to just do install